NETWORK/NetworkCreateSynchronisedScene.md

```diff
@@ -23,4 +23,26 @@ int NETWORK_CREATE_SYNCHRONISED_SCENE(float x, float y, float z, float xRot, flo
 * **animSpeed**: 
 
 ## Return value
-netScene id
+Network scene handle. This can be used to start / stop it and add peds or entities to it.
+
+## Examples:
+```lua
+    local ped = PlayerPedId()
+    local pedCo = GetEntityCoords(ped)
+    local sceneObject = GetClosestObjectOfType(pedCo, 2.0, GetEntityModel('prop_weed_01'), false, false, false)
+    local animDict = 'anim@gangops@facility@servers@bodysearch@'
+
+    RequestAnimDict(animDict)
+    while not HasAnimDictLoaded(animDict) do Wait(0) end
+
+    local netScene = NetworkCreateSynchronisedScene(GetEntityCoords(sceneObject), GetEntityRotation(sceneObject), 2, true, false, 1, 15.666667, 1.0)
+    NetworkAddPedToSynchronisedScene(ped, netScene, animDict, 'player_search', 1.0, 1.0, 15.666667, 17957384, 0) -- 15.666667 is the duration of this anim
+
+    NetworkStartSynchronisedScene(netScene)
+    Wait(15.666667)
+    NetworkStopSynchronisedScene(netScene)
+    ClearPedTasks(ped)
+```
+
+## Advanced example
+You can find an advanced example [here](https://pastebin.com/masLVdm5)
```

Comment:
This should be on the description, above parameters section. Also, 'double hash' (`##`) custom headers like these are not allowed, but 'triple hash' headers are. (`###`), so ideally `### Advanced example`.

It's also somewhat discouraged to use pastebin links since they can expire/get removed, causing the research to be lost.